### PR TITLE
Literally removes RnD from every ship

### PR DESCRIPTION
**Pull request by Apogee-dev**
as_we_speak

## About The Pull Request

Removes RnD from every ship, potentially as we speak.

## Why It's Good For The Game

Ships are too independent and the job of the scientist and medic is far too stale- All they do is stand in one place and click buttons. Removing RnD encourages cooperation and more interaction with the outpost cargo system.

## Changelog

:cl:
balance: Fixes RnD balance
/:cl:

### CODE REVIEW

**Review by rye-rice — APPROVED** [review]

**Review by Erikafox — APPROVED** [review]
I'm testmerging this tomorrow.
No one can stop me

**Comment by axelzonvolt** [comment]
this is going to make rogus cry

**Comment by meemofcourse** [comment]
shiptesting_season

**Comment by thgvr** [comment]
so called "ship freeze":

**Comment by MidnightLunarCat** [comment]
I will be honest, despite having not interacted with RnD much i would say you should probably add something to replace it so that you aren't giving people even less gameplay options then they already had, lately a lot of PRs have just been entirely removing content for balance reasons without providing any replacements and that honestly kind of sucks for the people that might have found the removed content fun. I have stated multiple times despite prefering RP over gameplay that shiptesters need more gameplay things to do, and removing gameplay (even if its poorly designed gameplay) without providing any new alternatives does basically the opposite of solve that problem.

**Comment by Ms-Mee** [comment]
I disagree with the changelog, as I am the known changelog disagreer (Pure evil)

**Comment by Merlin1230** [comment]
Honestly I dont really think removing early access from a system really fixes the problem of jobs being stale, since its just removing part of their job until they scavenge rnd. What if we did the exact opposite and gave all ships rnd? That would stop people from saying a ship is “bad” for not having it, and I also feel removing it doesnt really add.. anything to the small gameplay loop that is there

**Comment by meemofcourse** [comment]
it's funny though

**Comment by Ms-Mee** [comment]
The pill was a "funny" joke, look where we are now

**Comment by meemofcourse** [comment]
yes but the pill was merged this is a joke pr

**Comment by Ms-Mee** [comment]
Until I merge it :joker:

**Comment by Sun-Soaked** [comment]
Cast it into the fire, Frodo! Destroy it!

**Comment by Randomguy523** [comment]
As someone who does a lot of sci on Shiptest, but also believes it needs to be gutted to some degree, I fully agree with Merlin and Midnight on their takes; though I also believe the issue stems more from just how much the techfabs and design disks can make. Trimming down higher tier parts and tools rather than stalling the system entirely would likely prove more interesting, 
rather than removing it initially until someone lucks out on a ruin with an RnD setup (which then just resumes the RnD problems).

>the job of the scientist and medic is far too stale- All they do is stand in one place and click buttons. 

It is mainly because of the current systems that encourage little movement, and there's not a lot to easily change that fact without a rework of research almost as a whole.

On top of these, lack of changes to the alternatives (salvage, cargo prices), as well as no other changes to roundstart gear could further amplify some annoyances, and could make some ships feel even more lacking, while giving even less potential supplies to trade for initially.

TL;DR, RnD needs rework, not removal. Scarcity is good, but not like this. This will just encourage people to game for RnD ruins more than ever. Alternatives would be useful if this is the way we go.

**Comment by Erikafox** [comment]
Everyone remember the Pubby? It was a nice little compact ship with an old style mining redemption machine that people would hop on and create mining colonies with the unlimited shelter pods that invited its colonizers to stay and the ship to fly out into space and bring in points/supplies and create a nice unique little mining colony world with its own economy and community that worked without research and development which invited outsiders to bring in what it lacked. It also lacked medical supplies until you got points to use its medipens, causing brain damage or, if it was here today, would need external medical surgery tools. You had to be creative and come up with clever unique solutions to problems that the Pubby had. This made it fun to play it, as opposed to playing a ship with everything on it.

People rarely played the pubby then. And, when it was played, it was entertaining at least if people knew what they were doing. Once that old style mining redemption machine got removed and no way to get research to make a cargo board to buy more supplies, no one played it because there was nothing advantageous about it. So, the devs decided that to fix it they would make it bigger and give them a cargo lathe.

End result? No one plays Pubby, because they can just play solgov Cricket or some other ship with actual research and weaponry.

Another ship, sterilized. Another ship unplayed.

And now, a hunting ship that deals with dangerous space fauna is getting its big Magnum removed because of PVP. And, despite admins advocating for Roleplay over PVP, you're taking away a reasonable weapon that a fauna hunter would have in limited quantities to deal with giant space monsters. The gun is a point and click magnum, that could easily miss, and has a limited firing quantity because the bullets are indeed limited. There's no way to get ammo for this with research.... or anything really. And, even with it being as powerful as it is, you still need to use a substantial amount of its ammo to kill a person or a fauna, meaning you're going to go through the ammo pretty quickly, especially if you miss.

And, even with that, let's not forget that there is a common weapon known as the shotgun who's buckshot is a net for anyone to get hit with and has near unlimited supply if you have an autolathe. So, why remove it? Roleplay is already encouraged by admins, and PVP looked down upon. So why remove the fauna hunting/research vessel's unique gun. Just to take away its flavor? To sterilize it for.... who? PVPers? Then remove grenades, and infinite ammo, and infinite... guns too, since that's what research provides with the lathes. And, they're all more effective than the magnum.

I've been playing ss13 for almost 5 years, and a pattern I always notice is that roleplay correlates with gimmicks and fun. And, when you remove those gimmicks, and force people to interact with each other in a sterile environment, nobody wants to play with each other anymore. It's already happening, most ships on Ship test are either barren or have cliques that hunker down and avoid contact with other ships.

It's no wonder people had fun with the pill ship, if it was just you and another, you could just hop onto a pill ship and play, and it got removed because... there were too many pill ships because... most people wanted to play by themselves? This got solved by removing pill ship, and everyone just hopping onto big ships by themselves anyways.

And now... because the unique fauna hunting vessel has a fauna hunting gun, its getting removed because for some reason a big gun for big fauna is for some reason effective on smaller intelligent fauna if it hits. Is it not logical that a hunting ship with weapons for big fauna would also be dangerous to people? People that can create grenades that activate on word commands and blow up a quarter of your ship? Or, worse, have a shotgun that they have to aim in the general direction of you to hit you. Is a big magnum with a limited amount of bullets really that scary compared to the arsenal any other player can create within minutes?

God forbid I shoot someone with a hunting mangum when they could easily have a grenade spear/shotgun/grenade/MECH and it work.

**Comment by thgvr** [comment]
this is a meme btw we won't be doing this without making other things reasonable

**Comment by ZephyrTFA** [comment]
LGTM

**Comment by Erikafox** [comment]
firefox_ssayU757gG
image

**Comment by Apogee-dev** [comment]
Oh dear.

**Comment by Sun-Soaked** [comment]
> Oh dear.

image

My le pull request... was pulled?
